docs/features_es.md

```diff
@@ -6,30 +6,29 @@ lang: es
 
 # Caracteristicas de Scully
 
-Debajo del capó, Scully analiza su aplicación Angular y genera una versión estática
-de la misma. Proporciona varios schematics de Angular para hacer
-¡su uso TAN FÁCIL COMO ES POSIBLE!
+Tras bambalinas, Scully analiza la plicación de Angular y genera una versión estática de esta. Proporciona varios esquemas (schematics) de Angular para
+hacer su uso LO MÁS SENCILLO POSIBLE!
 
-[Caracteristicas Core](/docs/scully-lib-core_es):
+[Características Principales](/docs/scully-lib-core):
```

Comment:
Isra if you remove the `_es` in the links, this will we to redirect to the English documentation!

Reply:
Ok